src/components/Article.jsx

```diff
@@ -0,0 +1,27 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { getArticle } from "../api";
+
+const Article = () => {
+  const { article_id } = useParams();
+  console.log(article_id)
+  const [article, setArticle] = useState({});
+
+  useEffect(() => {
+    getArticle(article_id).then((article) => {
+        setArticle(article.data)
+    })
```

Comment:
We should have to have .data on this side, this should be the article object itself passed from the getArticle function